database/migrations/2016_03_25_234612_create_users_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->increments('id');
+            $table->timestamps();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::create('users', function(Blueprint $table)
```

Comment:
in function down there should be only a Schema::drop('tablename');
create should be in funciton up
function up is used when php artisan migrate is runned to create table in database
function down is used when php artisan migrate::reset to drop the table